### PR TITLE
feat(fe2): invite + list workspace invites

### PR DESCRIPTION
**Pull request by fabis94**

### CODE REVIEW

File: packages/frontend-2/components/project/page/automations/Header.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
         placeholder="Search automations..."
         wrapper-classes="shrink-0"
         show-clear
-        :model-value="bind.modelValue.value"
+        v-bind="bind"
```
Comment (fabis94) [inline packages/frontend-2/components/project/page/automations/Header.vue:13]: Improved `useDebouncedTextInput` return to allow for way simpler/easier configuration of the FormTextInput component

File: packages/frontend-2/components/settings/workspaces/members/TableHeader.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      <FormTextInput
+        name="search"
+        :custom-icon="MagnifyingGlassIcon"
+        color="foundation"
+        full-width
+        search
+        show-clear
+        :placeholder="searchPlaceholder"
+        class="rounded-md border border-outline-3"
+        v-bind="bind"
```
Comment (fabis94) [inline packages/frontend-2/components/settings/workspaces/members/TableHeader.vue:13]: Improved `useDebouncedTextInput` return to allow for way simpler/easier configuration of the FormTextInput component

File: packages/ui-components/src/components/layout/Dialog.vue [hunk 3]
```diff
@@ -92,7 +92,7 @@
               </div>
               <div
                 v-if="hasButtons"
-                class="relative z-50 flex px-6 py-3 space-x-3 shrink-0 bg-foundation-page border-t border-outline-2"
+                class="relative z-50 flex px-6 py-3 space-x-3 shrink-0 bg-foundation-page"
```
Comment (fabis94) [inline packages/ui-components/src/components/layout/Dialog.vue:95]: None of Michal's new dialogs have a border above the button footer

File: packages/ui-components/src/components/layout/Table.vue [hunk 4]
```diff
@@ -21,7 +20,17 @@
         class="divide-y divide-outline-3 h-full overflow-visible"
         :class="{ 'pb-32': overflowCells }"
       >
-        <template v-if="items.length">
+        <div
+          v-if="loading || !items"
```
Comment (fabis94) [inline packages/ui-components/src/components/layout/Table.vue:24]: Undefined `items` now means - show loading bar (for easy feeding in of API data)

File: packages/frontend-2/components/settings/workspaces/members/MembersTable.vue [hunk 5]
```diff
@@ -0,0 +1,157 @@
+<template>
```
Comment (fabis94) [inline packages/frontend-2/components/settings/workspaces/members/MembersTable.vue:1]: Moved+Updated Mike's table

File: packages/server/modules/workspaces/events/eventListener.ts [hunk 6]
```diff
@@ -41,13 +41,15 @@ export const onProjectCreatedFactory =
     const workspaceMembers = await getWorkspaceRoles({ workspaceId })
 
     await Promise.all(
-      workspaceMembers.map(({ userId, role: workspaceRole }) =>
-        grantStreamPermissions({
+      workspaceMembers.map(({ userId, role: workspaceRole }) => {
+        if (workspaceRole === Roles.Workspace.Guest) return false
```
Comment (fabis94) [inline packages/server/modules/workspaces/events/eventListener.ts:45]: cc @cdriesler

**Review by cdriesler — APPROVED** [review]

**Review by Mikehrn — CHANGES_REQUESTED** [review]
Nice 😁 Just one small thingy

File: packages/server/modules/workspaces/events/eventListener.ts (hunk 6 above)
Comment (fabis94) [reply packages/server/modules/workspaces/events/eventListener.ts:45]: @cdriesler There's no difference, the point is to return out. The return value is irrelevant, it'll just get awaited and then forgotten about.

I'd rather not add extra looping throughout the entire array just to filter out things. If you think the `return false` is confusing, I can make it a void return

**Review by Mikehrn — APPROVED** [review]

**Pull request merged**